### PR TITLE
Update TOC for DG,UG & minor touchups

### LINKED ISSUE #68: Fix UG hyperlinks
Need to cross-check the hyperlinked parts of the UG and make sure they work

### PR DESCRIPTION
**Pull request by rtshkmr**
Fixes #68 
TOC updates and cursory proofreading done. Need everyone to look through one last time before we can close v1.1

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1,8 +1,42 @@
 ---
 layout: page
-title: Travelling BusinessMan, TBM Developer Guide
+title: Travelling BusinessMan(TBM) Developer Guide
```
Comment (qwoprocks) [inline docs/DeveloperGuide.md]: I think we should just stick to either the abbreviation TBM or the full name in the headers, not both

Comment (qwoprocks) [reply docs/DeveloperGuide.md]: either that or perhaps change both back to the simple "User Guide" and "Developer Guide". I don't really see a need to have TBM in the title, as the whole website is already about TBM, plus we have introductions and our home page.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1,8 +1,42 @@
...
+- [**Appendix: Requirements**](#appendix-requirements)
+  - [Product scope](#product-scope)
+  - [User stories](#user-stories)
+  - [Use cases](#use-cases)
+  - [Non-Functional Requirements](#non-functional-requirements)
+  - [Glossary](#glossary)
+- [**Appendix: Instructions for manual testing**](#appendix-instructions-for-manual-testing)
+  - [Launch and shutdown](#launch-and-shutdown)
+  - [Deleting a person](#deleting-a-person)
+  - [Saving data](#saving-data)
```
Comment (tankangliang) [inline docs/DeveloperGuide.md]: The table of contents is actually generated by GitHub with 
```
* Table of Contents
{:toc}
```

With this addition we will have two copies of the TOC on our documentation website.

image

image

While editing this, could you also change the spelling of "Travelling Businessman" in "Glossary" to have a capitalized "M"?

**Review by tankangliang — COMMENTED** [review]
Links in UserGuide are working as expected. 👍

**Review by LeeEnHao — COMMENTED** [review]
Sorry my bad, while you're at it, is it alright to change the title tag of both Udes guide and dev guide to just 'User Guide' and 'Developer Guide'. The thing is showing up in full in the navbar of our main page. I added it without checking seeing the main page

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (rtshkmr) [reply docs/DeveloperGuide.md]: done, put it as  "User Guide" and "Developer Guide" for now

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (rtshkmr) [reply docs/DeveloperGuide.md]: done

**Review by raysonkoh — APPROVED** [review]
Looks good!

**Pull request merged**